app/src/pages/inside/common/suiteTestToolbar/actionPanel/actionPanel.jsx

```diff
@@ -71,11 +71,11 @@ const messages = defineMessages({
   },
   noBugTrackingSystemToLinkIssue: {
     id: 'ActionPanel.noBugTrackingSystemToLinkIssue',
-    defaultMessage: 'Configure bug tracking system to link issue',
+    defaultMessage: 'BTS integration is not configured. Connect with Admin',
```

Comment:
It repeats 4 times. I think it's better be moved to `common/constants/localization.js`